core/services/pipeline/runner.go

```diff
@@ -156,7 +181,15 @@ func (r *runner) Name() string {
 }
 
 func (r *runner) HealthReport() map[string]error {
-	return map[string]error{r.Name(): r.Healthy()}
+	cache, isCache := r.btORM.(*bridges.BridgeCache)
```

Comment:
Could we just check for the HealthReporter interface?

Reply:
I changed the check, but it still just checks `Ready()`. I would like to run `HealthReport()` but that returns a map and I don't know a clean way to combine maps other than just looping through both.

Reply:
`services.CopyHealth` is typically used https://github.com/smartcontractkit/chainlink-common/blob/main/pkg%2Fservices%2Fhealth.go#L18